<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Dae Kwan Ko | Michigan State University</title>
  
  <meta name="author" content="Dae Kwan Ko">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
	<link rel="icon" type="image/png" href="images/Michigan-State-Spartans-logo.png">

  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.css">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
</head>

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <strong><name>Dae Kwan Ko, Ph.D.</name></strong>
              </p>
              <p>I am an Assistant Professor - Fixed Term in <a href="https://prl.natsci.msu.edu">the Plant Research Laboratory</a>, <a href="https://plantbiology.natsci.msu.edu">the Department of Plant Biology</a>, and <a href="https://www.glbrc.org">the Great Lakes Bioenergy Research Center (GLBRC)</a> at Michigan State University (MSU).
              </p>
              <p>
                During my Ph.D. studies in <a href="https://molecularbiosci.utexas.edu">the Department of Molecular Biosciences at the University of Texas at Austin</a>, I mainly investigated <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006197">the molecular mechanism of hybrid vigor, known as heterosis, in maize using an integrative approach of experimental biology and genomics</a>.
                </p>
              <p>
                After completing my Ph.D. studies, as a postdoc, I joined <a href="https://buell-lab.github.io/robin.html">the lab of C. Robin Buell</a> at MSU for bioinformatics training. Following the successful training, I joined <a href="https://brandizzilab.natsci.msu.edu">the lab of Federica Brandizzi</a> where, as a postdoc, I investigated gene regulatory mechanisms underlying responses to endoplasmic reticulum stress in plants using systems-level approaches</a>. Particularly, I have developed coexpression network-based gene discovery pipelines and applied them to pinpointing regulatory hub genes for functional characterization.
              </p>
              	In my current position, I investigate how genes are dynamically regulated in plants, in response to climate change using multi-omics network approaches. My research is funded by GLBRC through <a href="https://brandizzilab.natsci.msu.edu">the Brandizzi Lab</a> and by <a href="https://www.canr.msu.edu/project-greeen/index">the MSU GREEEN project</a> for which I serve as a PI.
              </p>
              <p style="text-align:center">
              	<a href="mailto:dkko@msu.edu"><i
                  class="fa-solid fa-envelope"></i></a> &nbsp/&nbsp
                <a href="data/CV_DaeKwanKo.pdf"><i
                  class="ai ai-cv"></i></a> &nbsp/&nbsp
                <a href="https://github.com/DaeKwan-Ko/"><i
                  class="fa-brands fa-github"></i></a> &nbsp/&nbsp
                <a href="https://www.linkedin.com/in/daekwanko/"><i
                  class="fa-brands fa-linkedin"></i></a> &nbsp/&nbsp
                <a href="https://twitter.com/DaeKwan_Ko"><i
                  class="fa-brands fa-twitter"></i></a> &nbsp/&nbsp
                <a href="data/Dae Kwan Ko_bio.txt">Bio</a> &nbsp/&nbsp
                <a href="https://scholar.google.com/citations?hl=en&user=MXSf08oAAAAJ&view_op=list_works&sortby=pubdate">Google Scholar</a>
              </p>
            </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <a href="images/Dae Kwan Ko_lab.jpeg"><img style="width:90%;max-width:90%" alt="profile photo" src="images/Dae Kwan Ko_lab.jpeg" class="hoverZoomLink"></a>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
 
             <!-- Research interests-->
            <div id="research" style="margin-top: -50px;padding-top: 50px">
                <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;padding-top:50px; margin-top:-50px"><tbody>
                <tr>
                    <td style="padding:20px;width:100%;vertical-align:middle">
                        <strong><name>Research interests</name></strong>
                        <p>
               			 I am an early-career researcher who is passionate about addressing critical biological questions at the systems-level with a hypothesis-driven approach using the power of genomics. Do genes and proteins interact with each other in cells as we do in society? Then, what could be the functional consequences of those interactions in regulating biological pathways? I believe that these molecular interactions, called “biological networks”, are crucial for all living organisms to maintain cellular homeostasis. My long-term research goal is toward characterizing gene networks underlying significant biological pathways and applying the findings to translational research. Representative papers are <span class="highlight">highlighted</span> in yellow below.
             			</p>
                    </td>
                </tr>
                </tbody>
                </table>
            </div>

            <!-- Publication -->
            <div id="publications" style="margin-top: -50px;padding-top: 50px">
                <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                <tr>
                    <td style="padding:20px;width:100%;vertical-align:middle">
                        <strong><name>Publications</name></strong>
                    </td>
                </tr>
                </tbody></table>

                <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                    <tbody>
                    <!-- ism93 -->
                    <tr bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Ko et al_NatPlants2023.png">
                                    <img src='images/Ko et al_NatPlants2023.png' width="180" alt="ism93">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://www.nature.com/articles/s41477-023-01480-3">
                                <papertitle>An IRE1-proteasome system signaling cohort controls cell fate determination in unresolved proteotoxic stress of the plant ER</papertitle>
                            </a>
              				<br>
              				<strong>Ko DK</strong>, Kim JY, Thibault ET, Brandizzi F
             				<br>
              				Nat Plants. 2023 9:1333-1346.
              				<br>
              				<a href="https://github.com/DaeKwan-Ko/ism93">Github page for scripts</a> /
             		 		<a href="https://msutoday.msu.edu/news/2023/plant-biology-switch-between-life-and-death">Media highlight</a>
              				<p></p>
              				<p>
              				Description: In this study, we performed a forward genetic screening and discovered a new regulator that helps cells determine life or death under unresolved ER stress. Using an interdisciplinary approach of omics, biochemistry, and genetics, we identified its working mechanism and functional relationship with existing regulators.
              				</p>
            			</td>
                    </tr>  
		 
                    <!-- tgnap1 -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Bhandari et al_NatComms2023.png">
                                    <img src='images/Bhandari et al_NatComms2023.png' width="180" alt="tgnap1">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://link.springer.com/protocol/10.1007/978-1-0716-2784-6_27">
                                <papertitle>Defense against phytopathogens relies on efficient antimicrobial protein secretion mediated by the microtubule-binding protein TGNap1</papertitle>
                            </a>
              				<br>
              				Bhandari BD, <strong>Ko DK</strong>, Kim SJ, Nomura K, He SY, Brandizzi F
             				<br>
              				Nat Commun. 2023 14,6357.
              				<br>
              				<p></p>
              				<p>
              				Description: Defining plant defense machinery against pathogens is significant in cell biology and crop yield. This paper shows that TGNap1, a TGN and microtubule-binding protein, is required for defense and efficient anti-microbial protein secretion, linking secretion and cytoskeleton. I performed the RNA-seq analysis as a co-author. 
              				</p>
            			</td>
                    </tr>  

                    <!-- wgcna -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Ko and Brandizzi_Plant Proteostasis 2023_1.pdf">
                                    <img src='images/Ko and Brandizzi_Plant Proteostasis 2023_1.pdf' width="180" alt="WGCNA">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://link.springer.com/protocol/10.1007/978-1-0716-2784-6_27">
                                <papertitle>Coexpression Network Construction and Visualization from Transcriptomes Underlying ER Stress Responses</papertitle>
                            </a>
              				<br>
              				<strong>Ko DK</strong>, Brandizzi F
             				<br>
              				Plant Proteostasis. Methods in Molecular Biology, 2023 vol 2581. Humana, New York, NY
              				<br>
              				<a href="https://github.com/DaeKwan-Ko/WGCNA">Github page</a>
              				<p></p>
              				<p>
              				Description: The paper provides a step-by-step workflow for coexpression network analyses using WGCNA and discusses its potential applications.
              				</p>
            			</td>
                    </tr>  					
  
                     <!-- GBF2 -->
                    <tr bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Ko and Brandizzi_Nature Plants 2022_1.png">
                                    <img src='images/Ko and Brandizzi_Nature Plants 2022_1.png' width="180" alt="GBF2">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://www.nature.com/articles/s41477-022-01150-w#article-info">
                                <papertitle>Transcriptional competition shapes proteotoxic ER stress resolution</papertitle>
                            </a>
              				<br>
              				<strong>Ko DK</strong> and Brandizzi F
             				<br>
              				Nat Plants. 2022 May;8(5):481-490.
              				<br>
              				<a href="https://github.com/DaeKwan-Ko/UPR-TFs">Github page for scripts</a> /
             		 		<a href="https://prl.natsci.msu.edu/news-events/news/plant-er-advanced-genomics-illuminate-new-mechanisms-for-stress-mitigation/">Media highlight</a>
              				<p></p>
              				<p>
              				Description: We discovered a new UPR regulator and characterized its functional role in the UPR at the mechanistic level through a multi-omics approach.
              				</p>
            			</td>
                    </tr>  
		 
                     <!-- recovery network -->
                    <tr bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Ko and Brandizzi_Communications Biology 2022_1.png">
                                    <img src='images/Ko and Brandizzi_Communications Biology 2022_1.png' width="180" alt="recovery network">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://www.nature.com/articles/s42003-021-02964-8">
                                <papertitle>Advanced genomics identifies growth effectors for proteotoxic ER stress recovery in Arabidopsis thaliana</papertitle>
                            </a>
              				<br>
              				<strong>Ko DK</strong>, Brandizzi F
              				<br>
              				Commun Biol. 2022 Jan 11;5(1):16.
              				<br>
              				<a href="https://github.com/DaeKwan-Ko/UPR-GRN">Github page for scripts</a> /
              				<a href="https://prl.natsci.msu.edu/news-events/news/plants-get-stressed-too-were-looking-at-thousands-of-genes-to-understand-how-they-manage-it/">Media highlight</a>
              				<p></p>
              				<p>
              				Description: To better understand the transcriptional activities of the UPR master regulators, bZIP28 and bZIP60, I generated time-series transcriptome datasets and performed coexpression analyses, which allowed me to pinpoint downstream genes for functional characterization.
              				</p>
            			</td>
                    </tr> 

                     <!-- root review-->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/York et al_The Plant Phenome 2022_1.jpg">
                                    <img src='images/York et al_The Plant Phenome 2022_1.jpg' width="180" alt="root review">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
              				<a href="https://acsess.onlinelibrary.wiley.com/doi/abs/10.1002/ppj2.20028">
                				<papertitle>Bioenergy Belowground: challenges and opportunities for phenotyping roots and the microbiome for sustainable bioenergy crop production</papertitle>
              				</a>
              				<br>
              				York LM, Cumming JR, Trusiak A .... <strong>Ko DK</strong> .... Yang WH
              				<br>
              				The Plant Phenome Journal 5.1 2022: e20028.
              				<br>
              				<a href="https://www.glbrc.org">GLBRC website</a>
              				<p></p>
              				<p>
              				Description: In this white paper, the GLBRC colleagues and I discussed the significance and challenges of root biology in bioenergy research.
              				</p>
            			</td>
                    </tr>

                     <!-- pca-->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Jha et al_eLife 2021_2.jpg">
                                    <img src='images/Jha et al_eLife 2021_2.jpg' width="180" alt="pca">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
              				<a href="https://elifesciences.org/articles/66877">
                				<papertitle>Science forum: vision, challenges and opportunities for a plant cell atlas</papertitle>
              				</a>
              				<br>
              				Plant Cell Atlas Consortium, Ghosh Jha S, Borowsky AT .... <strong>Ko DK</strong> ... Rhee SY
              				<br>
              				eLife. 2021 Sep 7;10:e66877.
              				<br>
              				<a href="https://www.plantcellatlas.org">Plant Cell Atlas website</a>
              				<p></p>
              				<p>
              				Description: Plant Cell Atlas (PCA) colleagues and I reported the activities that we initiated for the plant research community.
              				</p>
            			</td>
                    </tr>
    
                     <!-- astrobiology -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Angelos et al_Astrobiology 2021_2.png">
                                    <img src='images/Angelos et al_Astrobiology 2021_2.png' width="180" alt="astrobiology">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://www.liebertpub.com/doi/abs/10.1089/ast.2020.2313">
                				<papertitle>Relevance of the Unfolded Protein Response to Spaceflight-Induced Transcriptional Reprogramming in Arabidopsis</papertitle>
              				</a>
              				<br>
              				Angelos E, <strong>Ko DK</strong>, Zemelis-Durfee S, Brandizzi F
              				<br>
              				Astrobiology. 2021 Mar;21(3):367-380.
              				<br>
              				<a href="https://prl.natsci.msu.edu/news-events/news/learning-what-it-takes-to-grow-a-space-garden/">Media highlight</a>
              				<p></p>
              				<p>
              				Description: This study focused on investigating transcriptome changes affected by UPR regulators in space. Plants were germinated, grown, and harvested in spaceflight. As a co-author, I performed RNA-seq analyses.
              				</p>
            			</td>
                    </tr>         

                     <!-- maize UPR -->
                    <tr bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Ko and Brandizzi_TPJ2021.png">
                                    <img src='images/Ko and Brandizzi_TPJ2021.png' width="180" alt="maize UPR">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/tpj.15044">
                				<papertitle>A temporal hierarchy underpins the transcription factor‐DNA interactome of the maize UPR</papertitle>
              				</a>
              				<br>
              				<strong>Ko DK</strong>, Brandizzi F
              				<br>
              				Plant J. 2021 Jan;105(1):254-270.
              				<br>
              				<a href="https://prl.natsci.msu.edu/news-events/research-highlights/identifying-transcription-factor-networks-that-protect-maize-when-its-main-cellular-biofactory-is-under-stress/">Media highlight</a>
              				<p></p>
              				<p>
              				Description: How the UPR is controlled is largely unknown in non-model plant species including maize. To better understand the regulatory landscape of the UPR, I performed TF network analyses based on enhanced Y1H screens for selected UPR marker genes in maize.
              				</p>
            			</td>
                    </tr> 				

                     <!-- pca report -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/PCA report.png">
                                    <img src='images/PCA report.png' width="180" alt="pca report">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/tpj.15044">
                				<papertitle>First plant cell atlas workshop report</papertitle>
              				</a>
              				<br>
              				Rice S, Fryer E, Ghosh Jha S...The Plant Cell Atlas Consortium (including <strong>Ko DK</strong>)
              				<br>
              				Plant Direct. 00:1–10.
              				<br>
              				<a href="https://www.plantcellatlas.org">Plant Cell Atlas website</a>
              				<p></p>
              				<p>
              				Description: In this white, Plant Cell Atlas (PCA) colleagues and I report the very first workshop that was successfully held online
              				</p>
            			</td>
                    </tr>
                    
                     <!-- network review -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Ko et al_TPJ_review_Figure1.pdf">
                                    <img src='images/Ko et al_TPJ_review_Figure1.pdf' width="180" alt="network review">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/tpj.14940">
                				<papertitle>Network‐based approaches for understanding gene regulation and function in plants</papertitle>
              				</a>
              				<br>
              				<strong>Ko DK</strong>, Brandizzi F
              				<br>
              				Plant J. 2020 Oct;104(2):302-317.
              				<br>
              				<a href="https://www.sebiology.org/magazine-publication/silico-in-biology/#p=28">Highlighted in the Society for Experimental Biology's Spring Bulletin</a>
              				<p></p>
              				<p>
              				Description: Understanding gene function is a critical requirement to advance knowledge of the principles underpinning fundamental and applied plant biology. In this review, we describe predictive analyses, their strengths and pitfalls that are fast-forwarding our understanding of gene regulation and function in plants.
              				</p>
            			</td>
                    </tr>
                    
                     <!-- UPR TIBS -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Pastor-Cantizano et al_TIBS 2019.jpg">
                                    <img src='images/Pastor-Cantizano et al_TIBS 2019.jpg' width="180" alt="UPR TIBS">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://www.sciencedirect.com/science/article/pii/S0968000419302099">
                				<papertitle>Functional diversification of ER stress responses in Arabidopsis</papertitle>
              				</a>
              				<br>
              				Pastor-Cantizano N, <strong>Ko DK</strong>, Angelos E, Pu Y, Brandizzi F.
              				<br>
              				Trends Biochem Sci. 2020 Feb;45(2):123-136.
              				<br>
              				<a href="https://reader.elsevier.com/reader/sd/pii/S096800041930252X?token=90BBCF78E442BD71E62B18F5EDFEBAB1FE18E1C678C1598DE4159EEFDADED681EA78A86820D424679B71622FB74824E8&originRegion=us-east-1&originCreation=20221209171105">On The Cover</a>
              				<p></p>
              				<p>
              				Description: In recent years, the field of plant UPR has substantially advanced, revealing new regulators and mechanisms. My colleagues and I provide recent updates on the new development in the plant UPR to the research community. 
              				</p>
            			</td>
                    </tr>                	
 
                      <!-- amplocon-seq -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Nadakuduti et al_FrotiersPlantScience 2019.jpg">
                                    <img src='images/Nadakuduti et al_FrotiersPlantScience 2019.jpg' width="180" alt="amplocon-seq">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://www.frontiersin.org/articles/10.3389/fpls.2019.00110/full">
                				<papertitle>Evaluation of Methods to Assess in vivo Activity of Engineered Genome-Editing Nucleases in Protoplasts</papertitle>
              				</a>
              				<br>
              				Nadakuduti SS, Starker CG, <strong>Ko DK</strong>, Jayakody TB, Buell CR, Voytas DF, Douches DS.
              				<br>
              				Front Plant Sci. 2019 Feb 8;10:110.
              				<br>
              				<p></p>
              				<p>
              				Description: We compare three existing methods for detection of gene editing activity in potato protoplasts. In this collaborative project, I performed amplicon-seq analyses as a co-author. 
              				</p>
            			</td>
                    </tr>
 
                      <!-- als1 -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Ko et al_2018 PlosOne.png">
                                    <img src='images/Ko et al_2018 PlosOne.png' width="180" alt="als1">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0206055">
                				<papertitle>Transcriptome profiling of transgenic potato plants provides insights into variability caused by plant transformation</papertitle>
              				</a>
              				<br>
              				<strong>Ko DK</strong>, Nadakuduti SS, Douches DS, Buell CR.
              				<br>
              				PLoS One. 2018 Nov 8;13(11):e0206055.
              				<br>
              				<p></p>
              				<p>
              				Description: We describe the impact of Agrobacterium-mediated transformation of a mutated acetolactate synthase gene on the transcriptome of potato, highlight the extent of positional effect of transgene insertion and that one component of somaclonal variation is due to altered expression of transcription factors and their downstream targets.
              				</p>
            			</td>
                    </tr>
                    
                      <!-- heterosis -->
                    <tr bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <div class="two"> 
								<video width="180" height="150" autoplay loop muted controls>
								<source src="images/10_29_2010.mp4" type=video/mp4>
								</video>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006197">
                				<papertitle>Temporal shift of circadian-mediated gene expression and carbon fixation contributes to biomass heterosis in maize hybrids</papertitle>
              				</a>
              				<br>
              				<strong>*Ko DK</strong>, *Rohozinski D, Song Q, Taylor SH, Juenger TE, Harmon FG, *Chen ZJ. *These authors contributed equally to this work
              				<br>
              				PLoS Genet. 2016 Jul 28;12(7):e1006197.
              				<br>
              				<p></p>
              				<p>
              				Description: Heterosis, or hybrid vigor, has been widely used in agriculture for more than a century. Despite extensive investigation and various models proposed, the molecular basis for heterosis remains largely elusive. I led this collaborative project to identify and characterize the molecular link of growth heterosis and the circadian clock. We report that higher carbon fixation and starch accumulation in maize hybrids than in the parents are associated with differences in the diurnal regulation of gene expression.
              				</p>
            			</td>
                    </tr>

                      <!-- allo -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Shi et al_2015 MBE.png">
                                    <img src='images/Shi et al_2015 MBE.png' width="180" alt="allo">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://academic.oup.com/mbe/article/32/9/2351/1029857">
                				<papertitle>Genome-wide dosage-dependent and-independent regulation contributes to gene expression and evolutionary novelty in plant polyploids</papertitle>
              				</a>
              				<br>
              				Shi X, Zhang C, <strong>Ko DK</strong>, Chen ZJ.
              				<br>
              				Mol Biol Evol. 2015 Sep;32(9):2351-66.
              				<br>
              				<p></p>
              				<p>
              				Description: We investigate allelic expression novelties using a series of newly synthesized Arabidopsis tetraploids that contain one, two, three, and four genomes of A. thaliana or A. arenosa, as well as reciprocal crosses containing the same nuclear genomes but different cytoplasms. 
              				</p>
            			</td>
                    </tr>
                    
                      <!-- bHLH -->
                    <tr #bgcolor="#ffffd0">
                        <td style="padding:20px;width:25%;vertical-align:middle">
                            <div class="one">
                                <a href="images/Ko et al_JPP 2009.jpg">
                                    <img src='images/Ko et al_JPP 2009.jpg' width="180" alt="bHLH">
                                </a>
                            </div>
                        </td>
                        <td style="padding:20px;width:75%;vertical-align:middle">
                            <a href="https://www.sciencedirect.com/science/article/pii/S0176161708003878">
                				<papertitle>Submergence-inducible and circadian rhythmic basic helix–loop–helix protein gene in Nicotiana tabacum</papertitle>
              				</a>
              				<br>
              				<strong>Ko DK</strong>, Lee MO, Hahn JS, Kim BG, Hong CB.
              				<br>
              				J Plant Physiol. 2009 Jul 1;166(10):1090-100..
              				<br>
              				<p></p>
              				<p>
              				Description: In my MS research, I identified and characterized a flooding stress-responsive transcription factor gene of which expression was not only responsive to flooding stress but also under the circadian clock regulation in Nicotiana. It demonstrated potential crosstalk between the clock and abiotic stress responses.
              				</p>
            			</td>
                    </tr>
                                                                              
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:0px">
              <br>
              <p style="text-align:right;font-size:small;">
              </p>
            </td>
          </tr>
        </tbody></table>
      </td>
    </tr>
  </table>
</body>

            <!-- Footer -->
            <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
                <tbody>
                <tr>
                    <td>
                        <br>
                        <p align="right">
                            <font size="2">
                                Website template courtesy of <a href="http://jonbarron.info/">Jon Barron.</a>
                            </font>
                        </p>
                    </td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
</table>
</body>

</html>